src/main/drivers/flash/flash_mt29f.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Author: jflyper
+ */
```

Comment:
New files need to use use https://github.com/betaflight/betaflight/blob/master/DEFAULT_LICENSE.md

```suggestion
/*

    This file is part of Betaflight.
    Betaflight is free software. You can redistribute this software
    and/or modify this software under the terms of the GNU General
    Public License as published by the Free Software Foundation,
    either version 3 of the License, or (at your option) any later
    version.
    Betaflight is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
    See the GNU General Public License for more details.
    You should have received a copy of the GNU General Public
    License along with this software.
    If not, see http://www.gnu.org/licenses/.
*/
```